frontend/packages/client/src/App.sass

```diff
@@ -582,7 +583,16 @@ span[data-tooltip]
 	-moz-transition: opacity 0.3s ease-out
 	-ms-transition: opacity 0.3s ease-out
 	-o-transition: opacity 0.3s ease-out
-	opacity: 0	
+	opacity: 0
+
+.fade-out-half
+	-webkit-transition: opacity 0.3s ease-out
+	-moz-transition: opacity 0.3s ease-out
+	-ms-transition: opacity 0.3s ease-out
+	-o-transition: opacity 0.3s ease-out
+	opacity: 0.5
+	&.is-disabled
+		pointer-events: none
```

Comment:
there wasn't a design for the "loading" state of the email signup process - eg after you click signup but before the request has finished, so I just reduce the opacity of the entire modal and disable click events to indicate a loading state